iocage/lib/Release.py

```diff
@@ -531,10 +532,11 @@ def fetch_updates(
                 if self.host.distribution.name == "FreeBSD":
                     with open(local_path, "r+") as f:
                         content = f.read()
+                        pattern = re.compile("^Components .+$", re.MULTILINE)
```

Comment:
my non-existent horse for a world where we can do string replacement without regex…